spec/requests/oidc_users_spec.rb

```diff
@@ -11,6 +11,11 @@
   let(:subject_identifier) { "subject-identifier" }
 
   describe "PUT" do
+    before do
+      next_oidc_user_id = ActiveRecord::Base.connection.execute("select last_value from oidc_users_id_seq").first["last_value"] + 1
```

Comment:
This seems like an odd query, can you unpack this line for me and why we need to do it this way?
I read that as:
```
- use an active record method to:
- grab the sequentially highest oidc user id?
- access the last value
- add one to it...
```

Why are we doing this instead of using  a factory / fixture or a mock?

Reply:
ah i see so we're getting the range and then going one beyond it to ensure there's never any clash?

Reply:
It's because the account-api user ID is passed as a parameter to email-alert-api, but we want to test the case where the account-api user doesn't exist yet.  Which means it'll create a new user than then call email-alert-api with that user's ID - so to stub the request, we need to know what that ID will be.